mrp_reapir_widget_digitized_signature/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================================
+Web Digital Signature for mrp repair
+=====================================
+
+This module adds a signature field to mrp repair form.
```

Comment:
"This module adds a digitized signature field to the MRP repair form 
so that technician repairman can have their customers sign their work reports."